Documentation/markdown/GitProcess.md

```diff
@@ -19,17 +19,33 @@ Before you can proceed to work with Git, there are several steps, which have to
 
 3. *Clone* your repository to local mashine.
 
-	- Open your File Explorer and select the folder, where you want to clone the OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
+	- Open your File Explorer and select the folder, where you want to clone OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
 
 	- In the window, which you have opened, in the command line write out *git clone https://github.com/<your_username>/Open-Infra-Platform.git* . This will start a download.
 
-4. Add remote: git add 
+4. Add remote: *git remote add tumcms https://github.com/tumcms/Open-Infra-Platform.git*
```

Comment:
Please add another sentence or two. Why "tumcms" as name, what does this command do, perhaps with a forward reference to the "Updating from main repo".

Reply:
Done